src/js/tabs/auxiliary.js

```diff
@@ -53,10 +53,15 @@ auxiliary.initialize = function (callback) {
 
         let modeName = FC.AUX_CONFIG[modeIndex];
         // Adjust the name of the box if a peripheral is selected
-        modeName = adjustBoxNameIfPeripheralWithModeID(modeId, modeName);
+        const modeNameAjusted = adjustBoxNameIfPeripheralWithModeID(modeId, modeName);
+        // remove blanks to use modeName as key in help index
+        modeName = modeName.replaceAll(' ', '');
```

Comment:
I suggest to use a new variable name and not reuse the `modeName`. And in others places where we "concat" for the messages files, we use the camelize method:
https://github.com/betaflight/betaflight-configurator/blob/8d5533bbd03781f68345532e80ff77ed8910104b/src/js/tabs/osd.js#L2714

Reply:
Changed